.gitignore

```diff
@@ -56,3 +56,4 @@ public/css/main*
 webpack-bundle-stats.html
 server/rev-manifest.json
 google-credentials.json
+/.vs
```

Comment:
What is this about?

Reply:
Visual Studio uses a hidden folder called .vs to store personal settings and preferences. I ignored it so it is not included in my push. I can push again using git bash if this is making issues.